core/injector/full_node_injector.cpp

```diff
@@ -0,0 +1,6 @@
+/**
```

Comment:
Is this file going to be needed?

Reply:
just to create full node injector target